README.md

```diff
@@ -15,17 +15,19 @@ Connect to the UID2 service, refresh the encryption keys, and then use the keys
 ```
 from uid2_client import Uid2Client, decrypt
```

Comment:
could you please update the python sdk readme.md here to follow the format and content of the [java sdk](https://github.com/IABTechLab/uid2-client-java)? Python sdk won't need the [Usage for Publishers](https://github.com/IABTechLab/uid2-client-java#usage-for-publishers) section but it should have other sections and replace the links to java example/codes to python's respective ones. 

you can reuse [here](https://unifiedid.com/docs/sdks/uid2-sdk-ref-python#usage-for-uid2-sharers) for Usage for UID2 Sharers section